.devcontainer/docker-compose.dev.yml

```diff
@@ -2,6 +2,7 @@ version: '3'
 
 services:
   web:
+    platform: linux/amd64
```

Comment:
I'm not sure what practical difference this is supposed to make? I've been building and running locally without it (on M1) without any trouble... 😕

Reply:
@stacyk Did you hit a particular issue with the Metecho build that required adding this? Reading the Docker docs, it sounds like this setting should only be used as a last resort. https://docs.docker.com/desktop/mac/apple-silicon/#known-issues

Reply:
@jgerigmeyer I think this was the first of three things I did to make it run. I didn't go back and remove it to see if the other two were fine on their own. We could certainly try and take it away and if I have a problem moving forward, I will be sure to let you know ;)

Reply:
Ok, my theory is that it might be needed on MetaDeploy because of the additional install_sfdx script? I'm going to try removing it here...